File: Chapter five/Python/compound_interest.py
import math

principal = 1000.0

for interest in range(5,11):
    interestConversion = interest / 100.0;

    print("This is compound interest for " , interest , " rate")
    print(f"{'year':<10} {'Amount on Deposit':>5}")

    for year in range(1,10):
        amount = principal * math.pow(1.0 + interestConversion, year)
        print(f"{year:<10} {amount:>5}" )

File: Chapter five/Java/CompoundInterest.java

public class CompoundInterest{
public static void main(String[] args){

double principal = 1000.0;



for(int interest = 5; interest <= 10;interest++){

double interestConversion = interest / 100.0;

System.out.println("This is compound interest for " + interest + " rate");

System.out.printf("%s%20s%n", "Year", "Amount on deposit");
for (int year = 1; year <= 10;year++) {

double amount = principal * Math.pow(1.0 + interestConversion, year);

System.out.printf("%4d%,20.2f%n", year, amount);
}

}




}
}
